src/dvclive/dvc.py

```diff
@@ -177,3 +181,29 @@ def get_dvc_stage_template(live):
     dvcyaml = output.getvalue()
     output.close()
     return dvcyaml
+
+
+def update_models(path, attrs, models):
+    from dvc.repo.artifacts import name_is_compatible
+
+    if attrs["name"] is None:
+        name = find_unused_name(path, models)
```

Comment:
I think we can drop `find_unused_name`. If users are worried about conflicts, they can explicitly set a name.

Reply:
I included it because it was explicitly mentioned as a requirement in our last sync with @aguschin and @omesser .
I am happy to go ahead and remove code if there are no strong objections